contributors/devel/controllers.md

```diff
@@ -103,6 +103,30 @@ func (c *Controller) Run(threadiness int, stopCh chan struct{}) {
 		go wait.Until(c.runWorker, time.Second, stopCh)
 	}
 
+	// register event handlers to fill the queue with pod creations, updates and deletions
+	c.pods.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
this should happen during construction, not in Run(). adding the event handler here (after start) would be ignored

Reply:
@liggitt it works after Run as been called. See [here](https://github.com/kubernetes/client-go/blob/db8228460e2de17f5d3a9a453f61dde0ba86545a/tools/cache/shared_informer.go#L318).